packages/manager/modules/netapp/src/module.js

```diff
@@ -3,6 +3,8 @@ import angular from 'angular';
 import '@ovh-ux/manager-core';
 import '@uirouter/angularjs';
 import 'angular-translate';
+import ngAtInternet from '@ovh-ux/ng-at-internet';
```

Comment:
We tend to import AngularJS modules this way (to access to its module name):

```js
import '@ovh-ux/ng-at-internet';
```

Is there any reason here to prefer using a named import?

Reply:
Just copying what README.md says (see https://github.com/ovh/manager/blob/master/packages/components/ng-at-internet/README.md#Usage)